en/docs/reference/connectors/kafka-connector/kafka-connector-producer-example.md

```diff
@@ -24,29 +24,63 @@ Follow these steps to set up the Integration Project and the Connector Exporter
 
 {!includes/reference/connectors/importing-connector-to-integration-studio.md!}
 
-5. Create a new Kafka connection by selecting a particular operation.
+7. Specify the API name as `KafkaTransport` and API context as `/publishMessages`. 
 
-    <a href="{{base_path}}/assets/img/integrate/connectors/filecon10.png"><img src="{{base_path}}/assets/img/integrate/connectors/filecon10.png" title="working directory" width="800" alt="working directory"/></a>
+8. To configure the resource click on the API Resource and go to **Properties** view. Select the method POST.
 
+9. Next drag and drop the 'publishMessages' operation of the KafkaTransport Connector to the Design View as shown below.
+   <a href="{{base_path}}/assets/img/integrate/connectors/kafka"><img src="{{base_path}}/assets/img/integrate/connectors/kafka/kafka-add-connector.png" title="Add publishMessages connector operation" width="800" alt="Add publishMessages connector operation"/></a>
 
-1. Right click on the created Integration Project and select **New** -> **Rest API** to create the REST API.
+10. Create a connection from the properties window by clicking on the '+' icon as shown below.
 
-2. Specify the API name as `KafkaTransport` and API context as `/publishMessages`. You can go to the source view of the XML configuration file of the API and copy the following configuration (source view).
+    <a href="{{base_path}}/assets/img/integrate/connectors/kafka"><img src="{{base_path}}/assets/img/integrate/connectors/kafka/kafka-add-connection.png" title="Create Connection" width="800" alt="Create Connection"/></a>
 
+    In the popup window, the following parameters must be provided.
+- Bootstrap Servers - The Kafka brokers listed as host1:port1 and host2:port2.
+- Key Serializer Class - The serializer class for the key that implements the serializer interface.
+- Value Serializer Class - The serializer class for the value that implements the serializer interface.
+
+    <a href="{{base_path}}/assets/img/integrate/connectors/kafka"><img src="{{base_path}}/assets/img/integrate/connectors/kafka/kafka-connection-configuration.png" title="Connection Configuration" width="800" alt="Connection Configuration"/></a>
```

Comment:
This screenshot should be under Avro Message example

Reply:
Fixed with 1889736aed7fbdd77eb868c72dfc10b8f7049056